src/EventSubscriber/CacheableResponseSubscriber.php

```diff
@@ -90,13 +90,23 @@ public function onRespond(ResponseEvent $event) {
       }
 
       $tags_string = implode(' ', $tags);
-      if (25000 < strlen($tags_string)) {
-        $tags_string = substr($tags_string, 0, 25000);
-        // The string might have cut of in the middle of a tag.
-        // So now find the the last occurence of a space and cut to that length.
+      // value also defined in config install and schema files
+      $limit = 25000;
+
+      // Allow to lower the surrogate key header limit on non-Pantheon Env
+      if (!isset($_ENV['PANTHEON_ENVIRONMENT'])) {
```

Comment:
```suggestion
      if (!isset($_ENV['PANTHEON_ENVIRONMENT'])  ||
    $_ENV['PANTHEON_ENVIRONMENT'] === 'lando') {
```
I am always conflicted about this, Lando is popular enough with our customers that we should accommodate its use of PANTHEON_ENVIRONMENT within the Pantheon recipe.

Reply:
Is this another instance where PANTHEON_INFRASTRUCTURE_ENVIRONMENT would be more appropriate to check?

Reply:
that's a good alternative too.